code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -70,6 +70,37 @@
 
 	material = "wood"
 
+/obj/item/stack/tile/wood/proc/build(turf/S as turf)
+	if(istype(S,/turf/unsimulated))
+		S.ChangeTurf(/turf/simulated/floor/plating/deck/airless)

```

Comment:
Why are you making this an airless simulated turf, sure it makes sense for asteroids maybe but lots of unsim turfs have air